tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -31,6 +31,7 @@
 #include <TError.h>
 #include <TFile.h>
 #include <TKey.h>
+#include "ROOT/RPage.hxx"
```

Comment:
```suggestion
#include <ROOT/RPageAllocator.hxx>
```
(and in the right place please)

Reply:
Sorry, this was automatically added by my editor - a feature I hate and I was sure I managed to disabled, but apparently not...